lib/routes/builds.js

```diff
@@ -314,28 +319,105 @@ module.exports = function (app) {
    *             type: object
    *     responses:
    *       $ref: '#/responses/Standard'
+   *
+   *   put:
```

Comment:
i believe we have a different path then defined above due to `env` but I cant remember how annoying it is with APIG to do optional parameters or if it should work as is. We should ask @msluther

Reply:
swagger doesn't actually support optional path parameters, so you end up having to define `/foo/bar` and `/foo` both (assuming `bar` is optional)